source/thisTarget.ts

```diff
@@ -61,7 +61,7 @@ export function getTabDataStatus(): typeof tabDataStatus {
 }
 
 const storeTabData = once(async () => {
-  if (tabDataStatus !== "needed") {
+  if (isBackground()) {
```

Comment:
We don't want to skip fetching in every case, just if we are in the background service worker (since we want to refetch when the prerendered tab is activated.)

Reply:
If this change is necessary, it should be reflected in the `tabDataStatus`, not here. Also it seems to be duplicated in the "call again callback"